features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -104,11 +124,54 @@ The PSA on single v7-M is supported with the help of Mbed OS PSA services. The f
 Please pay attention to config options `extra_labels_add` and `device_has_add`. If needed then a PSA target definition **MUST** use [extra_labels/device_has]`_add` or [extra_labels/device_has]`_remove` (not `extra_labels` or `device_has`) to add/remove either extra_labels or target capabilities. Also, use `[feature_]`add/remove to add/remove a feature.
 Check [extra_labels](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html), [device_has](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) and [features](https://os.mbed.com/docs/mbed-os/v5.14/reference/adding-and-configuring-targets.html) for more information.
 
-### Adding dual v7-M targets
+## Adding dual v7-M targets
+A target can ge categorized as a dual v7-M target if it has at least two Armv7-M cores. On dual v7-M targets, TF-M will be running on SPE providing PSA services. Therefore, it is **MANDATORY** to define both SPE and NSPE target for dual v7-M targets.
+
+The SPE target **MUST** contain following attributes,
+
+* inherits: PSA generic target PSA_DUAL_V7_M_SPE
+* tfm_target_name: Target name in TF-M
+* tfm_bootloader_supported: If TF-M bootloader is supported by the target. Values supported are "true" and "false"
+* tfm_default_toolchain: Default TF-M toolchain supported. Values supported are "ARMCLANG" and "GNUARM"
```

Comment:
@Patater The toolchain names `ARMCLANG` and `GNUARM` are used by TF-M. Therefore new Mbed OS targets that supports TF-M should have these names.

Other option is to continue using Mbed OS toolchain names and use key-value pair in the python script to identify TF-M toolchain names. Since the toolchains are target specific configuration I'd prefer define them in targets.json.

Reply:
OK, I am happy to use TF-M names then. The consumer is the TF-M build system. Best to match that.